lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/controller/GraphController.java

```diff
@@ -241,6 +248,29 @@ private void initListeners() {
                                         "Graph not loaded while attempting to add known mutations.")));
                     } else {
                         try {
+                            insertKnownMutations((File) args[0]);
+                        } catch (IOException exception) {
+                            shout(NotificationController.NOTIFY, "",
+                                    notFact.getNotification(exception));
+                        }
+                    }
+                });
+
+        listen(Message.OPENED,
```

Comment:
You don't need listen(Message.OPENED) 3 times, you can combine those to a single one.